documentation/proposals/Proposal - Multi-Backend Input.md

```diff
@@ -834,6 +831,103 @@ public interface IJoystickInputHandler : IInputHandler
 
 `HandleHatMove` must be called when any value of `JoystickState.Hats` changes.
 
+# Input Maps
```

Comment:
This should be a separate proposal given that there's no interest in having this as a core part of input, rather an extension to the existing input library. This also means we don't need to implement it for 3.0, rather we can do so at anytime in 3.X.